code/style.css

```diff
@@ -1,7 +1,211 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  font-family: 'Source Sans Pro', sans-serif;
+  background-color: rgb(243, 215, 178);
+}
+
+
+.grid-form {
+  display: flex;
+  flex-flow: column;
```

Comment:
I was wondering why you used flex-flow here, should it be flex-direction instead? It works anyway but with flex-flow you can define flex-direction and flex-wrap at the same time, for example flex-flow: column wrap;